datastructures/Board.cpp

```diff
@@ -35,11 +41,10 @@ bool Board::isMoveValid(Position &initPos, Position &destPos, Color player) {
         if (position == initPos)
             canBeDone = true;
     }
-    if (!canBeDone)
```

Comment:
I wrote this intentionally because there is code to be added after the if statement

Reply:
Good point, I'll revert that part!